### PR TITLE
Merge Application Error scripts

### PR DESCRIPTION
**Pull request by ManosMagnus**
Both scripts perform relevant scans.

### CODE REVIEW

**Comment by thc202** [comment]
Thank you!

File: addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ OPCODE_TEXT = 0x1;
 RISK_MEDIUM = 2;
 CONFIDENCE_MEDIUM = 2;
 
-var strings_microsoft_db_error = [/Microsoft OLE DB Provider for ODBC Drivers/igm,
+var microsoftDbErrors = [/Microsoft OLE DB Provider for ODBC Drivers/igm,
                                            /Microsoft OLE DB Provider for SQL Server/igm,
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js]: Not majorly important, there might be others too, this one just really stuck out with them side by side.

[the remark this answers is not in the document]
Comment (ManosMagnus) [reply addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js]: Not sure tbh. Seems to you that can cause any FP? if not we can merge them

[the remark this answers is not in the document]
Comment (thc202) [reply addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js]: If we change here we should change in the xml file too.

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js]: OK let's leave it.

File: addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js [hunk 2]
```diff
@@ -10,12 +10,12 @@ OPCODE_TEXT = 0x1;
...
                                            /ODBC Microsoft Access Driver/igm,
                                            /ODBC SQL Server Driver/igm];
 
-var strings_msql_error = [/supplied argument is not a valid MySQL result/igm,
+var msqlDbErrors = [/supplied argument is not a valid MySQL result/igm,
```
[the remark this answers is not in the document]
Comment (ManosMagnus) [reply addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js]: fixed

**Pull request merged**